src/core/include/units/bits/external/hacks.h

```diff
@@ -61,6 +62,7 @@
 #define UNITS_DIAGNOSTIC_IGNORE_NON_TEMPLATE_FRIEND
 #define UNITS_DIAGNOSTIC_IGNORE_SHADOW UNITS_DIAGNOSTIC_IGNORE(4459)
 #define UNITS_DIAGNOSTIC_IGNORE_UNREACHABLE UNITS_DIAGNOSTIC_IGNORE(4702)
+#define UNITS_DIAGNOSTIC_IGNORE_FLOAT_EQUAL
```

Comment:
These should come after `UNITS_DIAGNOSTIC_IGNORE_EXPR_ALWAYS_TF`.

Reply:
Done